### ISSUE TITLE
e-prop learning for MNIST classification.

### ISSUE THREAD

**Issue by LiYuan-0709** [ChFrenkel/eprop-PyTorch#1]
I tried to apply this PyTorch-implemented e-prop algorithm to the MNIST classification task. However, I only got an accuracy of 54%. I want to know what I should do to improve it for this digit classification task. Or, what I should do to make it adjust to a new task quickly because there are so many hyperparameters?
